website/search/elastic_search.py

```diff
@@ -418,11 +416,10 @@ def serialize_contributors(node):
     return {
         'contributors': [
             {
-                'fullname': user.fullname,
-                'url': user.profile_url if user.is_active else None
-            } for user in node.visible_contributors
-            if user is not None
-            and user.is_active
+                'fullname': x['user__fullname'],
+                'url': '/{}/'.format(x['user__guids___id'])
+            } for x in
+            node.contributor_set.filter(visible=True, user__is_active=True).order_by('contributor___order').values('user__fullname', 'user__guids___id')
```

Comment:
`.include()` is an order-of-magnitude speedup:

Without include:

```
In [1]: from django.core.paginator import Paginator

In [2]: paginator = Paginator(Node.objects.all(), 20)

In [3]: page = paginator.page(1)

In [4]: node = page[0]

In [5]: %time list(node.contributor_set.all())
CPU times: user 3.32 ms, sys: 1.26 ms, total: 4.57 ms
Wall time: 6.69 ms
Out[5]:
[<Contributor(user=jeswes@gmail.com, read=True, write=True, admin=True, visible=True)>,
 <Contributor(user=livsolis@utexas.edu, read=True, write=True, admin=True, visible=True)>,
 <Contributor(user=n.hutcheson@austin.utexas.edu, read=True, write=True, admin=True, visible=True)>]
```

With include:

```python
In [1]: from django.core.paginator import Paginator

In [2]: paginator = Paginator(Node.objects.all().include('contributor__user'), 20)

In [3]: page = paginator.page(1)

In [4]: node = page[0]

In [5]: %time list(node.contributor_set.all())
CPU times: user 143 µs, sys: 13 µs, total: 156 µs
Wall time: 152 µs
Out[5]:
[<Contributor(user=jeswes@gmail.com, read=True, write=True, admin=True, visible=True)>,
 <Contributor(user=livsolis@utexas.edu, read=True, write=True, admin=True, visible=True)>,
 <Contributor(user=n.hutcheson@austin.utexas.edu, read=True, write=True, admin=True, visible=True)>]
```